docs/metadao/faq.md

```diff
@@ -0,0 +1,35 @@
+# Frequently Asked Questions (FAQ)
+
+## 1. Futarchy Questions
+
+### Q: Will futarchy always be correct?
+**A:** Sometimes futarchy may result in decisions that seem suboptimal in hindsight. However, it remains one of the most effective governance mechanisms available.
+
+### Q: Ok this is amazing, should everything be governed by a futarchy?
+**A:** Not necessarily, see Vitalik's [blog post](https://vitalik.eth.limo/general/2022/09/20/daos.html) on DAO types.
```

Comment:
I think pretty much everything should be :)

Reply:
Agreed. Removed this question in PR